cpp/models/abm/person.h

```diff
@@ -68,6 +69,15 @@ class Person
         return (m_person_id == other.m_person_id);
     }
 
+    /**
+     * @brief The TestResult of a Person.
+     */
+    struct TestResult {
+        TimePoint time_of_testing; ///< The TimePoint when the Person performs the test.
+        TestType type = TestType::Count; ///< The TestType of the test.
```

Comment:
Probably not needed to initialise this here. The other two data members are also not initialised.

Reply:
I removed it now since I realised we have this information from the `m_test_results`.